src/actionlib/SimpleActionServer.js

```diff
@@ -185,6 +185,29 @@ SimpleActionServer.prototype.sendFeedback = function(feedback2) {
     this.feedbackPublisher.publish(feedbackMessage);
 };
 
+/**
+*  Set action state to aborted and return to client
+*/
+
+SimpleActionServer.prototype.setAborted = function(result2) {
+    
+
+    var resultMessage = new Message({
+        status : {goal_id : this.currentGoal.goal_id, status : 4},
```

Comment:
what's 4? is there an enum we could use or if not maybe we can just add a comment

Reply:
so according to my understanding, `4` is the code for ABORTED state but I took this code from upstream. so if I add enum I should/need to add to the whole code.